commands/logs.py

```diff
@@ -0,0 +1,53 @@
+
+from errors import MalformedMoYaml
+from errors import UnknownEnvironment
+from subprocess import call
+import click
+import sys
+import yaml
+import io
+
+
+def command_logs(env):
+    """Executes the heroku logs command with the tail flag. The default
+    environment is staging but production can be tailed using the environment
+    flag
+    """
+    try:
+        # Open the mo.yml file
+        # TODO: Search recursively until we get to the root of the project
+        stream = io.open('mo.yml', 'r')
```

Comment:
You can use the built-in context manager to close the io stream here.

```
with io.open('mo.yml', 'r') as stream:
    data = yaml.load(stream)
```

This will automatically call the `close()` method on the stream at the end of the context.